workspace/TS100/src/main.cpp

```diff
@@ -926,7 +926,7 @@ void startPIDTask(void const *argument __unused) {
 #ifdef MODEL_TS80
 	idealQCVoltage = calculateMaxVoltage(systemSettings.cutoutSetting);
 #endif
-	int32_t rawC = ctoTipMeasurement(100) - ctoTipMeasurement(101);  // 1*C change in raw.
+	uint8_t rawC = ctoTipMeasurement(101) - ctoTipMeasurement(100);  // 1*C change in raw.
```

Comment:
Why did you change this back to the uint8_t, would it not be better to let it stay as a int32_t, and then handle the overflow later on ?

Reply:
It's not a bounding issue, but due to casting a negative number into a uint8_t.

The overflow was from 100-101, which is -75 or so. That maps to 180 when cast to a uint8_t (I think?).

Anyway, it was messed up but within the bounds of a uint8_t.

Reply:
The fix actually shrunk the rom by about 20-30 bytes!

Reply:
Okay thats interesting, and GCC is running way above my head then 👍